upf/grpc/upf.proto

```diff
@@ -0,0 +1,63 @@
+syntax = "proto3";
+
+message Fluent {
+    string name = 1;
+    string valueType = 2;
+    repeated string signature = 3;
+}
+
+message Object {
+    string name = 1;
+    string type = 2; 
+}
+
+message Expression {
+    int64 type = 1;
+    repeated Expression args = 2;
+    Payload payload = 3;
+}
+
+message Assignment {
+    Expression x = 1;
+    Expression v = 2;
+}
+
+message Payload {
+    string type = 1;
+    string value = 2;
+}
+
+message Action {
```

Comment:
The `Action` name is probably to generic given that it is limited to classical/instantaneous actions.
Here me might want to have a unified view of an action or split it into different subclasses (InstantaneousAction, DurativeAction).